apps/server/lib/one-drive/src/internal/errors.ts

```
@@ -0,0 +1,49 @@
+				// @ts-ignore - not all envs have native support for cause yet
+				const error = new Error(
+					(err as { message?: string }).message ?? "",
+					(err as { cause?: unknown }).cause ? { cause: (err as { cause?: unknown }).cause } : {}
+				);
+
+				if ((err as { stack?: string }).stack) error.stack = (err as { stack?: string }).stack;
+				if ((err as { cause?: unknown }).cause && !("cause" in error)) {
+					// @ts-ignore
+					error.cause = (err as { cause?: unknown }).cause;
+				}
+				if ((err as { name?: string }).name) error.name = (err as { name?: string }).name!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)